tests/CourseCompletedTest.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace Tests;
+
+class CourseCompletedTest extends TestCase {}
```

Comment:
We implemented the course completed event too. I assumed I needed to put this file too along with the other test files that correspond to the other events. Shall I remove this file?